.gitlab-ci.yml

```diff
@@ -368,3 +369,15 @@ deploy:
       - SCRIPT: install_script_vector0.sh
   script:
     - $S3_CP_CMD ./${SCRIPT} s3://dd-agent/scripts/${SCRIPT} --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
+  after_script:
+    # invalidate the install.datadoghq.com CF distribution
+    - export $(printf "AWS_ACCESS_KEY_ID=%s AWS_SECRET_ACCESS_KEY=%s AWS_SESSION_TOKEN=%s"
```

Comment:
Can you actually do multiline commands like this?! I thought you either had to use `- >` or `- |` or at least enclose the string in a single/double quote :o
What a wonderful and clear syntax :sweat_smile:

Reply:
Yes you can, the yaml parser will connect them into a single oneliner string. The `- >` and `- |` are for when you need multiple lines.